server/JsDbg.Gdb/JsDbg.py

```diff
@@ -200,21 +243,18 @@ def LookupField(module, type, field):
         fields = [f for m in match for f in m.type.fields()]
 
 def LookupGlobalSymbol(module, symbol):
-    # We can't use lookup_global_symbol because that does not work for symbols
-    # with local linkage, such as those in an anonymous namespace.
-    # https://sourceware.org/bugzilla/show_bug.cgi?id=24474
-    (sym, _) = gdb.lookup_symbol(symbol)
+    sym = FindGdbSymbol(module, symbol)
```

Comment:
Does this get slower now that we incur the additional perf cost of gdb.objfiles()?

Reply:
Same for the calls to FindGdbType above/below.

Reply:
gdb.objfiles is pretty quick. I think this should be a net win because we now no longer call lookup_symbol when the objfile isn't loaded, and especially on GDB 8.4 (current Git, or whenever it will be released), where we only search one objfile instead of all of them, which is the slow part.

Reply:
Ok. SGTM.